src/Datadog.Trace/Propagators/IPropagator.cs

```diff
@@ -0,0 +1,21 @@
+namespace Datadog.Trace.Propagators
+{
+    /// <summary>
+    /// Interface for objects that can propagate a <see cref="SpanContext"/>
+    /// by injecting it into or extracting from a "wire" protocol, like HTTP.
+    /// </summary>
+    public interface IPropagator
```

Comment:
I don't understand the purpose of the `IPropagator` interface, it doesn't seem to be used anywhere.

Reply:
I simplified the code (https://github.com/DataDog/dd-trace-dotnet/pull/267/commits/3b982c7927a4d68aa8c3ffe7c121df51bbda95b3) to use only the extension methods instead of creating propagator instances.